packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -974,21 +974,19 @@ class VmServiceWrapper implements VmService {
   @override
   Future<Success> setTraceClassAllocation(
     String isolateId,
-    String classId,
+    String? classId,
```

Comment:
the parameter types in our wrapper class should match the parameter types from VmService. Make this non-nullable and then fix errors upstream at the call sites

Reply:
done